src/main/kotlin/com/openlattice/chronicle/controllers/TimeUseDiaryController.kt

```
@@ -135,19 +138,64 @@ class TimeUseDiaryController(
         return submissionsIdsByDate
     }
 
+    @Override
+    override fun downloadTimeUseDiaryData(
+        @PathVariable(ORGANIZATION_ID) organizationId: UUID,
+        @PathVariable(STUDY_ID) studyId: UUID,
+        @PathVariable(PARTICIPANT_ID) participantId: String,
+        @RequestParam(DOWNLOAD_TYPE) downloadType: TimeUseDiaryDownloadDataType,
+        @RequestBody submissionIds: Set<UUID>
+    ): Iterable<Map<String,Any>> {
+        return timeUseDiaryManager.downloadTimeUseDiaryData(
```

You can also just throw an Exception here as it should never used.